real-time/real-time.js

```diff
@@ -353,9 +353,9 @@ module.exports = connect.behavior("real-time",function(baseConnection){
 						var msg = "One or more items were retrieved which do not match the 'Set' parameters used to load them. "
 							+ "Read the docs for more information: http://v3.canjs.com/doc/can-set.html#SolvingCommonIssues"
```

Comment:
can we change this to point to https://canjs.com/doc/can-set.html#SolvingCommonIssues?